### PR TITLE
bug after copy webhook called for non translatable pages

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
Still needs tests

**Comment by codiebeulaine** [comment]
@miltontony test added

File: molo/core/wagtail_hooks.py [hunk 1]
```diff
@@ -47,6 +47,11 @@ def show_main_language_only(parent_page, pages, request):
 
 @hooks.register('after_copy_page')
 def copy_translation_pages(request, page, new_page):
+
+    # Only copy translations for TranslatablePageMixin
+    if not hasattr(page.specific, 'copy_language'):
+        return 'Not translatable page'
```
Comment (miltontony) [inline molo/core/wagtail_hooks.py]: @codiebeulaine why do you need to do this? Technically it shouldn't show anything at all. It should just continue as normal

**Comment by miltontony** [comment]
👍 🍰

**Pull request merged**